src/careamics/config/support/supported_data.py

```diff
@@ -82,3 +82,28 @@ def get_extension(cls, data_type: Union[str, SupportedData]) -> str:
             return "*.*"
         else:
             raise ValueError(f"Data type {data_type} is not supported.")
+
+    @classmethod
+    def get_extension(cls, data_type: Union[str | SupportedData]) -> str:
+        """
+        Get file extension of corresponding data type.
+
+        Parameters
+        ----------
+        data_type : str or SupportedData
+            Data type.
+
+        Returns
+        -------
+        str
+            Corresponding extension.
+        """
+        if data_type == cls.ARRAY:
+            raise NotImplementedError(f"Data {data_type} are not loaded from file.")
```

Comment:
Ok, this comment is probably going to much into the details... Shouldn't it be "files" plural? (I'll trust the native speaker there, just my feelings). Please disregard if disagree!

Reply:
I copy and pasted this from the original function above.

https://github.com/CAREamics/careamics/blob/6ff1d4b54f6491e71cc3360148ea946f76ccef1a/src/careamics/config/support/supported_data.py#L78

I think I will change both to "Data {data_type} _is_ not loaded from a file."

Reply:
Sounds good!

The code base is probably riddled with approximate english. 😄